### PR TITLE
Helpers in its own mod followup

### PR DESCRIPTION
**Pull request by grantseltzer**
This is a follow up to #246 which moved the `helpers` package into its own module.

In this PR, the helpers package imports the latest version of libbpfgo (commit at tip of main) because that's the first version that recognizes it as its own module.

It updates usage in the selftests, and removes the `libbpfgo` module's go.sum file because it doesn't actually import anything anymore!

### CODE REVIEW

File: selftest/tracing/go.mod [hunk 1]
```diff
@@ -1,12 +1,19 @@
-module github.com/aquasecurity/libbpfgo/selftest/perfbuffers
+module github.com/aquasecurity/libbpfgo/selftest/tracing
 
-go 1.18
+go 1.19
```
[the remark this answers is not in the document]
Comment (geyslan) [reply selftest/tracing/go.mod:3]: Related: https://github.com/aquasecurity/libbpfgo/pull/246/files#r987073629

[the remark this answers is not in the document]
Comment (grantseltzer) [reply selftest/tracing/go.mod:3]: @rafaeldtinoco Do you have an opinion on the matter? Upgrading libbpfgo to go 1.19 doesn't seem to have side effects in tracee. Though I do want tracee to upgrade.

**Comment by grantseltzer** [comment]
I'm investigating the failing unit tests. The `go.work` file fixes it for me locally but it's not in actions.

**Review by geyslan — APPROVED** [review]
LGTM! 🚀

**Pull request merged**